### PR TITLE
#161348787 Configure Authors Haven to use PostgreSQL as DB Engine

### PR DESCRIPTION
**Pull request by judeinno**
#### What does this PR do?
Configure Authors Haven to use postgreSQL as DB Engine

#### Description of Task to be completed?
Currently, the database used is sqlite
This task configures the app to use postgresSQL

#### How should this be manually tested?
- Pull the branch
- Create a postgres DB
- Add the DB parameters to a .env file(follow the example template(.env.example) at the root directory)
- Run migration commands `python manage.py makemigrations` and `python manage.py migrate`
- The tables will be created in your DB

#### Any background context you want to provide?
- The db uses python decouple library, the links below explain it better.
https://pypi.org/project/python-decouple/
https://simpleisbetterthancomplex.com/2015/11/26/package-of-the-week-python-decouple.html

#### What are the relevant pivotal tracker stories?
[#161348787](https://www.pivotaltracker.com/story/show/161348787)

### CODE REVIEW

**Comment by brucemakallan** [comment]
We need to brush up on our git workflow. This commit (https://github.com/andela/ah-backend-thanos/pull/3/commits/bacd63ba3af4f778d806543d00e657fc2ccbb219) has `develop` being merged into a chore branch. Let's have a quick walkthrough of git on Slack.

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SECRET_KEY=s3cr3t
```
Comment (emugaya) [inline .env.example]: Use environment variables. @judeinno

**Review by emugaya — CHANGES_REQUESTED** [review]
Address my comments.

File: .env.example (hunk 1 above)
Comment (judeinno) [reply .env.example]: I used environment variables that was just an example file that the team would use as a guide

Comment (emugaya) [reply .env.example]: Add examples to `ReadMe`.

**Pull request merged**